README.md

```diff
@@ -1,36 +1,46 @@
 # Java Analytics for Snowplow
 
-[![early-release]][tracker-classificiation] [![Build][github-image]][github] [![Release][release-image]][releases] [![License][license-image]][license]
+[![early-release]][tracker-classification] [![Build][github-image]][github] [![Release][release-image]][releases] [![License][license-image]][license]
 
 ## Overview
 
 Add analytics to your Java software with the **[Snowplow][snowplow]** event tracker for **[Java][java]**. See also: **[Snowplow Android Tracker][snowplow-android-tracker]**.
 
 With this tracker you can collect event data from your Java-based desktop and server apps, servlets and games. Supports JDK7+.
```

Comment:
I think we should change `Supports JDK7+` to `Supports JDK8+` too.
Because in build.gradle we have the constraint to `targetCompatibility = '1.8'`.
All these versions cause a huge headache!

Reply:
Done